code/game/gamemodes/cult/runes.dm

```diff
@@ -133,6 +133,11 @@ var/list/sacrificed = list()
 				target.hallucination = min(target.hallucination, 500)
 			return 0
 
+		//If you dont have blood, blood magic doesnt work on you
+		if(target.species.flags & IS_MECHANICAL)

```

Comment:
isSynthetic is a better check here, since you can convert carbon types to cult as well.